scripts/generate_seo_tags.rb

```
@@ -0,0 +1,232 @@
+#!/usr/bin/env ruby
```

RuboCop (1.73)

[warning] Script file generate_seo_tags.rb doesn't have execute permission.

(Lint/ScriptPermission)